### PR TITLE
Decision making statements/ It-academy.by / JC1-110-22 / Dmitry Vechorko / Done tasks 1-5

### PR DESCRIPTION
**Pull request by DmitryVechorko**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [ ] fixed
- [ ] approved

### CODE REVIEW

**Comment by DmitryVechorko** [comment]
feat: completed tasks 1-5

File: src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task01.java [hunk 1]
```diff
@@ -38,6 +38,11 @@ public static void main(String[] args) {
     static boolean isTopPlayer(int sizeTop, int positionPlayerByDamage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+        boolean topPlayer;
+        if (positionPlayerByDamage <= sizeTop) {
+            topPlayer = true;}
+        else {
+            topPlayer = false;}
+        return topPlayer;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task01.java:46]: "Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"

File: src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java [hunk 2]
```diff
@@ -37,6 +37,18 @@ public static void main(String[] args) {
...
+        } else if (hour >=6 && hour <=11) {
+            greeting = "Good morning";
+        } else if (hour >=12 && hour <=17) {
+            greeting = "Good day";
+        } else if (hour >=18 && hour <=23) {
+            greeting = "Good evening";
+        } else {
+            greeting = "Incorrect Hour";
+        }
+        return greeting;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java:52]: "Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"

Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java:52]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java [hunk 3]
```diff
@@ -37,6 +37,20 @@ public static void main(String[] args) {
...
+        String colorHealthPoint;
+        float currentHealthLevel;
+        currentHealthLevel = currentHealthPoint * 100 / maxHealthPoint;
+        if (currentHealthLevel >=0 && currentHealthLevel <25) {
+            colorHealthPoint = "RED";
+        } else if (currentHealthLevel >=25 && currentHealthLevel <50) {
+            colorHealthPoint = "ORANGE";
+        } else if (currentHealthLevel >=50 && currentHealthLevel <=75) {
+            colorHealthPoint = "YELLOW";
+        } else if (currentHealthLevel >=75 && currentHealthLevel <=100) {
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java:49]: "Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix and add another tasks

**Pull request closed without merging**